app/services/check_answers/sections/dependants.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+module CheckAnswers
+  module Sections
+    class Dependants < Base
+      def initialize(check)
+        super(check:, label: :dependants)
+      end
+
+      def subsections
+        child_deps = FieldPresenter.new(table_label: :dependant_details, attribute: :child_dependants, type: :boolean,
+                                        model: @check, partner_dependant_wording: true)
+        child_dep_count = if @check.child_dependants
+                            FieldPresenter.new(table_label: :dependant_details, attribute: :child_dependants_count,
+                                               type: :number_or_text,
+                                               model: @check)
+                          end
+        adult_deps = FieldPresenter.new(table_label: :dependant_details, attribute: :adult_dependants, type: :boolean,
+                                        model: @check, partner_dependant_wording: true)
+        adult_dep_count = if @check.adult_dependants
+                            FieldPresenter.new(table_label: :dependant_details, attribute: :adult_dependants_count,
+                                               type: :number_or_text,
+                                               model: @check)
+                          end
+
+        table_data = {
+          dependant_details:
+            [
+              child_deps,
+              child_dep_count,
+              adult_deps,
+              adult_dep_count,
+            ].compact,
+          dependant_income: [
+            FieldPresenter.new(table_label: :dependant_income, attribute: :dependants_get_income, type: :boolean,
```

Comment:
Need a conditional here to display this table if there is a child/adult dependant. Otherwise we are presented this. See the additional table marked in red. 

![Screenshot 2024-08-12 at 13 43 42](https://github.com/user-attachments/assets/7c7393e2-a911-43e9-bfb0-03e67457366b)